chats.py

```diff
@@ -13,6 +14,7 @@ def lasi():
 
 
 def pieraksti_zinju(dati):
+    json_data = dati["chats"]
+    json_data["laiks"] = datetime.now()
```

Comment:
Domāju, ka ar laiku būs par maz, man prasītos tomēr, lai ir arī datums.
Jo slack, piemēram, uzbīdot ar peli virsū laikam parādās arī datums.
Kā Tu domā?

Reply:
Vismaz man gan pārlūkā, gan .txt šis atgriež gan laiku, gan datumu.

Reply:
Nebiju uzmanīgs, nepiestartēju projektu pirms tam.